features/steps/room_steps.js

```diff
@@ -15,9 +18,16 @@ When("a {actor} provides the correct {room} Key", function (actor, room) {
   return "pending";
 });
 
-Then("the {actor} is placed in the {room}", function (actor, room) {
-  // Write code here that turns the phrase above into concrete actions
-  return "pending";
+When('the {actor} taps the {room} in the Room Picker', async function (actor, room) {
+  const workspacePage = new WorkspacePage(this.driver)
+  const roomCard = workspacePage.findRoomCard(room);
+  await workspacePage.enterRoom(roomCard);
```

Comment:
From a design perspective, I would probably remove a couple of the function calls here, maybe going with something like: 

```
When(‘the {actor} taps the {room} in the Room Picker’, async function (actor, roomName) {  
  this.workspace.enterRoom({ roomName })
})
```

This would require us to set `this.workspace` earlier on, perhaps in the `Given` .